tests/net/checksum_offload/prj.conf

```diff
@@ -33,3 +33,12 @@ CONFIG_NET_DEBUG_IPV4=n
 CONFIG_NET_DEBUG_NET_PKT=y
 CONFIG_SYS_LOG_NET_LEVEL=4
 CONFIG_NET_SHELL=n
+
+# Disable internal ethernet drivers as the test is self contained
+# and does not need the on board driver to function.
+CONFIG_ETH_NATIVE_POSIX=n
```

Comment:
We have no way if disabling ethernet completely, for all devices? for example CONFIG_ETH=n?

Reply:
Not that I know of. And beside we do not want to disable ethernet as that would disable the test, we just want to disable on board ethernet chip if the test is run on such board that has ethernet port.

Reply:
ok